### DISCUSSION TITLE
-morphooptiini error 2

### DISCUSSION THREAD

**Discussion by joshuajaistein** [neperfepx/neper#435] [Q&A] [answered]
Even on randomizing the seeds for my 5:5:1 cube I am unable to generate the geometry.I am trying to develop a 5:5:1 cube with 40 grains
[neperinput.txt](https://github.com/rquey/neper/files/8461346/neperinput.txt)
.

COMMANDS USED : 

for i in 1 2 3
do
  (gsl-randist $i 40 flat -1e-6 1e-6 > rand$i)
done
paste neperinput.txt rand? | awk '{print $1+$4,$2+$5,$3+$6}' neperinput.txt > neperinput.txt2

neper -T -n 40 -morphooptiini "coo:file(neperinput.txt2)" -domain "cube(5,5,1)" -periodicity all -format tesr,vtk -tesrsize 50:50:10 -o jjgeometry

**Comment by rquey** [comment, marked as answer]
Randomize more... It works with `1e-5` instead of `1e-6`.
